examples/react/start-supabase-basic/app/utils/supabase.ts

```diff
@@ -0,0 +1,25 @@
+import { parseCookies, setCookie } from 'vinxi/http'
+import { createServerClient } from '@supabase/ssr'
+
+export function useSupabase() {
```

Comment:
Please rename this util so it doesn't clash with the naming of React hooks. Maybe `getSupabaseServer`?

Reply:
Oops! Was trying to follow the previous example but that was indeed returning a hook. Changed!